tools/ansible/tasks/wsu/main.yaml

```diff
@@ -257,7 +257,8 @@
       win_get_url:
         url: "https://api-int.{{ cluster_address }}:22623/config/worker"
         dest: "{{ win_temp_dir.path }}\\worker.ign"
-        http_agent: "Ignition/0.35.0"
+        headers:
+          Accept: "application/vnd.coreos.ignition+json`;version=3.1.0"
```

Comment:
Is there an extra single quote after `ignition+json` ?

Reply:
\` is the escape character in PowerShell (I had to use it above at https://github.com/openshift/windows-machine-config-bootstrapper/pull/223/files#diff-631087f953268c655ad0d05d71b2f6e2R206 to escape `;`

That said, I'm not sure how or whether this ansible module might do some sanitation on input strings - it might not be necessary. Is there a way to test this code? (IIRC the wsu e2e test was removed a while back)

Reply:
I've removed this part since the playbook is going to be deprecated anyway (as per yesterday's decision in the design meeting)